src/server/core/shared/route-utils.ts

```
@@ -0,0 +1,243 @@
+): Promise<{
+  originDistances: { stationId: string; distance: number }[];
+  destinationDistances: { stationId: string; distance: number }[];
+  minOriginDistance: number;
+  minDestDistance: number;
+  closestOriginStation: string;
+  closestDestStation: string;
+}> {
+  const originDistances: { stationId: string; distance: number }[] = [];
+  const destinationDistances: { stationId: string; distance: number }[] = [];
+
+  let minOriginDistance = Infinity;
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)